src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/DefaultJavaScriptEncoderBasicLatin.cs

```diff
@@ -166,15 +183,26 @@ public override unsafe int FindFirstCharacterToEncode(char* text, int textLength
 
                     // Load the next 8 characters + a dummy known that it must not be escaped.
                     // Put the dummy second, so it's easier for GetIndexOfFirstNeedToEscape.
-                    Vector128<sbyte> sourceValue = Sse2.PackSignedSaturate(
-                        Sse2.LoadVector128(ptr),
-                        Vector128.Create((short)'A'));  // max. one "iteration", so no need to cache this vector
+                    Vector128<sbyte> sourceValue;
+
+                    if (Sse2.IsSupported)
+                    {
+                        sourceValue = Sse2.PackSignedSaturate(
+                            Sse2.LoadVector128(ptr),
+                            Vector128.Create((short)'A'));  // max. one "iteration", so no need to cache this vector
+                    }
+                    else
+                    {
+                        Debug.Assert(AdvSimd.Arm64.IsSupported);
```

Comment:
> Debug.Assert(AdvSimd.Arm64.IsSupported); [](start = 23, length = 41)

convert to `else if (AdvSimd.IsSupported)` ?

Reply:
Damn, missed that.